docs/wmco-prerequisites.md

```diff
@@ -52,5 +52,8 @@ consistent across all supported cloud providers.
   
 * User-Provisioned Infrastructure is NOT supported.
 
+## Supported Storage 
```

Comment:
This is not a pre-requisite but a limitation. I suggest adding a `Storage` sub-section under [limitations](https://github.com/openshift/windows-machine-config-operator#limitations)